exchanges/bithumb/bithumb_wrapper.go

```diff
@@ -228,11 +228,13 @@ func (b *Bithumb) UpdateOrderbook(p currency.Pair, assetType asset.Item) (orderb
 	}
 
 	for _, bids := range orderbookNew.Data.Bids {
```

Comment:
Out of scope, but I've noticed you've been improving these by using an index instead of a bids/asks var